### DISCUSSION TITLE
Disable mouse in fzf

### DISCUSSION THREAD

**Discussion by Yaco** [zk-org/zk#70] [Help] [answered]
Hi! I would like to disable the mouse interaction when invoking ```fzf``` and be able to select text with the mouse. ```fzf``` already got an option for that, "--no-mouse", but I don't know how to make ```zf``` to call ```fzf``` with that option.

Can you help me? Thanks!

**Comment by mickael-menu** [comment, marked as answer]
There's no way to customize `fzf` flags in [`zk` configuration](https://github.com/mickael-menu/zk/blob/main/docs/tool-fzf.md) yet, but that would be an interesting feature. You could modify directly the code if you want: https://github.com/mickael-menu/zk/blob/977625bb3d6728efdd7ea8d2376aed238af85972/internal/adapter/fzf/fzf.go#L75-L88

But the most `fzf`-y way to do that would be to set `FZF_DEFAULT_OPTS`, I think. This works for me:

```sh
$ FZF_DEFAULT_OPTS="--no-mouse" zk list -i
```
